test/org/sagebionetworks/bridge/validators/SurveySaveValidatorTest.java

```diff
@@ -881,11 +881,11 @@ public void operatorsInInfoScreenRulesAreValidated() {
         info.setPromptDetail("prompt detail");
         info.setIdentifier("identifier");
         info.setGuid("guid");
-        info.setRules(Lists.newArrayList(
+        info.setAfterRules(Lists.newArrayList(
                 new SurveyRule.Builder().withValue("foo").withOperator(Operator.EQ).withEndSurvey(true).build()));
```

Comment:
I see the test for validateOneRuleSet() and validateSkipToTargetExists(), but not validateOnRuleSetInInfoScreen().